go.mod

```diff
@@ -16,3 +16,5 @@ require (
 	knative.dev/networking v0.0.0-20220504043559-8d5dcbeeea2a
 	knative.dev/pkg v0.0.0-20220504135159-0719b405eca5
 )
+
+replace knative.dev/networking => github.com/skonto/networking v0.0.0-20220511140406-b9394caa78f2
```

Comment:
This should be removed once https://github.com/knative/networking/pull/668 is merged.